### PR TITLE
Lobby-db documentation, update prod and dev deployment notes

### PR DESCRIPTION
**Pull request by DanVanAtta**
## Overview
The lobby-db documentation had the pre-infrastructure way to deploy
prod. It's now updated to mention the important pieces in infrastructure
that will deploy DB changes.

Checking how infrastructure deploys, it
[installs FlyWay](https://github.com/triplea-game/infrastructure/blob/master/roles/application/lobby_db/tasks/flyway.sh)
 and executes all commands it needs. The prod deployment scripts removed
here are the old way to deploy prod and are no longer needed.

Second focus of this update is remove any developer notes to
install a local DB and instead focus on documenting the usage
of a docker container that hosts the DB and runs flyway for us.

## Functional Changes
- none expected, documentation update and legacy script removal

## Manual Testing Performed
- none

## Additional Review Notes
- pls note the infrastructure link to the fly way task file that runs flyway migration. With that we can pretty confidently delete the old prod deployment scripts
- the mention to use only docker and not a local setup of lobby-db is significant and is intended for us to make heavier use of docker containers to launch DB and potentially other components in the future (eg: dice-roller, http-lobby)

### CODE REVIEW

**Comment by RoiEXLab** [comment]
@DanVanAtta I can't follow the infrastructure flyway part. Should this PR get merged right now?

File: lobby-db/README.md [hunk 1]
```diff
@@ -2,139 +2,70 @@
...
-| Versioned | 1.04    | audit mutes                         | SQL  | 2018-03-14 21:14:11 | Success |
-| Versioned | 1.05    | add all user info to bans and mutes | SQL  | 2018-03-14 21:14:11 | Success |
-| Versioned | 1.06    | add access log                      | SQL  | 2018-03-14 21:14:11 | Success |
-+-----------+---------+-------------------------------------+------+---------------------+---------+
+- Production database changes are triggered by checking in
+  [flyway changes](https://github.com/triplea-game/triplea/tree/master/lobby-db/src/main/resources/db/migration)
+  followed by checking in an updated version number to infrastructure
+  [host_control.sh](https://github.com/triplea-game/infrastructure/blob/master/roles/host_control.sh)
 
+- Following that, when the [infastructure master branch] is merged to [infrastructure prod branch], then
```
Comment (ssoloff) [inline lobby-db/README.md]: Were these intentionally meant to be links?  If so, the URI needs to be included.

Comment (DanVanAtta) [reply lobby-db/README.md]: Good catch, looks like I never revisited those. They've been turned into links in: 0a1b4a2ca

**Comment by DanVanAtta** [comment]
> @DanVanAtta I can't follow the infrastructure flyway part. Should this PR get merged right now?

No, please point out anything that is confusing or hard to understand. I pushed an update that rewrites the content for better clarity: dee3035d4

Let me know how much that helped, and if things look complete and (at least somewhat) easy to understand   :grin:

**Comment by RoiEXLab** [comment]
👍
> pls note the infrastructure link to the fly way task file that runs flyway migration. With that we can pretty confidently delete the old prod deployment scripts

When would that be the case?

**Comment by DanVanAtta** [comment]
@RoiEXLab , I am slightly unsure of your question and may be confused

The 'infrastructure link' is noted as I looked at that to see the steps taken to do prod flyway deployment. I did not spot any dependencies on the scripts in 'triplea-game/triplea/lobby-db'. The infrastructure lobby-db is live, so the 'triplea/lobby-db' prod-scripts are already legacy.

**Comment by RoiEXLab** [comment]
@DanVanAtta Ahh makes sense. I didn't realise the lobby-db was live already. 
Thanks for clarifying

File: lobby-db/README.md [hunk 2]
```diff
@@ -1,140 +1,61 @@
...
-sudo apt install psql
-
-
-### Install docker
-
-- we'll use docker to launch a local postgres DB. Steps for that are given below in the [Docker](#docker) section.
-
-
-### How to use for developers
+### pre-requirements:
```
Comment (ssoloff) [inline lobby-db/README.md]:
```suggestion
### Prerequisites
```

**Pull request merged**